### DISCUSSION TITLE
3D model for desktop frame for 3.5" Turing screen

### DISCUSSION THREAD

**Discussion by majormer** [mathoudebine/turing-smart-screen-python#469] [General]
I needed somewhere to hold my Turing screen that was a bit more stable than my folding holder that came with the device. I created a model for a horizontal frame for the 3.5" Turing screen that slides in from the top. Working pretty well so far. I would create one of the other sizes, but unfortunately I don't have any examples to measure, etc.

If you are interested in 3D printing it (or having a service print it for you), you can find the model at [Thingiverse](https://www.thingiverse.com/thing:6487961)

![20240215_221313](https://github.com/mathoudebine/turing-smart-screen-python/assets/7697997/58d17373-9dd6-4d4f-b81d-dd8d82dba63f)

**Comment by ooemir23** [comment]
WOWW Niceee

**Comment by mathoudebine** [comment]
Thanks for sharing! ❤️  I will pin this discussion

**Comment by majormer** [comment]
I just purchased a 5" model today, so should have that model to create a frame for. Once I get that done, I will edit the top to add a link to that.

**Comment by DJRoby19** [comment]
Thank you so much for sharing
here is my 3d print
![20240310_230617](https://github.com/mathoudebine/turing-smart-screen-python/assets/132849164/bf7b289e-438b-4b31-96f0-6095f41c06fa)

**Reply by majormer** [reply]
I love it! Glad it worked for you!

**Reply by majormer** [reply]
I updated the Thingiverse link to include a 5" version and a new 3.5" version of the frame. Both new models have thin left sides. The reason is that having a USB-C cable jutting out the side is pretty obnoxious, so I purchased a right angle USB adapter that would send the cable to the back. Unfortunately with the original model, the frame was too thick to allow the right angle connector, so I flattened down that side.

Enjoy!
